components/CompanyLogos.js

```diff
@@ -1406,3 +1406,27 @@ export const DeadspinLogo = () => (
     ></path>
   </svg>
 );
+
+export const MiuiLogo = () => (
```

Comment:
I would personally go with the general MiUi Logo instead of using the one for MiUi 12. That's obviously if you're actually able to find it.